### PR TITLE
Adding Armenian translation

### PR DESCRIPTION
**Pull request by naregjan**
Added Armenian translation (`README.am.md`) and added link in `README.md`

### CODE REVIEW

**Comment by yuta0801** [comment]
no need all links in `README.am.md` and please add the link in `README.ja.md`

> Only README.md and README.ja.md contain the full list of the links to other languages, while other language versions should only have a link to English version.
https://github.com/hamukazu/lets-get-arrested/wiki#translation

**Comment by naregjan** [comment]
Did those things!

**Comment by naregjan** [comment]
Bump; can we get this merged in?

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-[English](README.md) | [日本語](README.ja.md) | [한국어](README.ko.md) | [Bahasa Indonesia](README.in.md) | [Español](README.es.md) | [Deutsch](README.de.md) | [français](README.fr.md) | [العربية](README.ar.md) | [Português Brasileiro](README.pt-br.md) | [Italiano](README.it.md) | [Nederlands](README.nl.md) | [简体中文](README.zh_hans.md) | [繁體中文](README.zh_hant.md) | [Türkçe](README.tr.md) | [বাংলা](README.bn.md)  | [Русский](README.ru.md) | [Polski](README.pl.md) | [Esperanto](README.eo.md) | [Svenska](README.se.md) | [Magyar](README.hu.md) | [Norsk](README.no.md) | [Bahasa Malaysia](README.ms.md) | [ภาษาไทย](README.th.md) | [Tiếng Việt](README.vi.md) | [Česky](README.cz.md) | [ئۇيغۇر](README.ug.md) | [Български](README.bg.md)
+
```
Comment (yuta0801) [inline README.md]: Please remove extra line breaks.

File: README.ja.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
-[English](README.md) | [日本語](README.ja.md) | [한국어](README.ko.md) | [Bahasa Indonesia](README.in.md) | [Español](README.es.md) | [Deutsch](README.de.md) | [français](README.fr.md) | [العربية](README.ar.md) | [Português Brasileiro](README.pt-br.md) | [Italiano](README.it.md) | [Nederlands](README.nl.md) | [简体中文](README.zh_hans.md) | [繁體中文](README.zh_hant.md) | [Türkçe](README.tr.md) | [বাংলা](README.bn.md)  | [Русский](README.ru.md) | [Polski](README.pl.md) | [Esperanto](README.eo.md) | [Svenska](README.se.md) | [Magyar](README.hu.md) | [Norsk](README.no.md) | [Bahasa Malaysia](README.ms.md) | [ภาษาไทย](README.th.md) | [Tiếng Việt](README.vi.md) | [Česky](README.cz.md) | [ئۇيغۇر](README.ug.md) | [Български](README.bg.md)
+
```
Comment (yuta0801) [inline README.ja.md]: Please remove extra line breaks.

File: README.am.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+[English](README.md)
+# Ձերբակալվելւո ծրագիր
```
Comment (yuta0801) [inline README.am.md]: Please add a line break between the link and title like any others.

**Pull request merged**